R/utilities-dimensions.R

```diff
@@ -0,0 +1,162 @@
+#' Dimension existence
+#'
+#' @param dimension String name of the dimension.
+#' @details Check if the provided dimension is supported.
+#' @export
+existsDimension <- function(dimension) {
+  enumHasKey(dimension, enum = Dimensions)
+}
+
+#' Validate dimension
+#'
+#' @param dimension String name of the dimension.
+#' @details Check if the provided dimension is supported. If not, throw an error
+#' @export
+validateDimension <- function(dimension) {

```

Comment:
should not we have one method (validateDimension or existDimension with a flag instead of two?) we have the same concept already somewhere else